backend/Dockerfile

```diff
@@ -71,22 +71,22 @@ COPY chimp_chomp/chimp.onnx /chimp.onnx
 COPY --from=build /chimp_chomp_libraries/* /lib
 COPY --from=build /app/target/release/chimp_chomp /chimp_chomp
 
-ENTRYPOINT ["./chimp_chomp"]
+ENTRYPOINT ["./backend/chimp_chomp"]
```

Comment:
Have you tested this? If not, give `podman build --target chimp_chomp` and `podman run <image_id>` a go